teuthology/suite.py

```diff
@@ -59,6 +60,10 @@ def main(args):
     filter_in = args['--filter']
     filter_out = args['--filter-out']
 
+    subset = None
+    if args['--subset']:
+        subset = tuple(map(int, args['--subset'].split('/')))
```

Comment:
Is this just making sure that each subset is an int and then making a tuple of them?  If so, maybe leave a comment explaining what this does as it's initially difficult to parse.